### PR TITLE
Gruppe 5

### PR DESCRIPTION
**Pull request by ins305**

### CODE REVIEW

File: TInf.g4 [hunk 1]
```diff
@@ -24,7 +24,7 @@ doWhileLoop: DO LBRACE stmtLst RBRACE WHILE LPAREN logicalExpr RPAREN SEMICOLON;
 forLoop: FOR LPAREN assignStmt SEMICOLON logicalExpr SEMICOLON logicalExpr RPAREN LBRACE stmtLst RBRACE;
 
 // Switch statement (team 5)
-switchStmt: SWITCH LPAREN logicalExpr RPAREN LBRACE cases default? RBRACE;
+switchStmt: SWITCH LPAREN IDENTIFIER RPAREN LBRACE cases default? RBRACE;
```
Comment (marcauberer) [inline TInf.g4:27]: Why that?

**Review by marcauberer — COMMENTED** [review]
The helpers on the AST node are missing. I left one other question. Otherwise lgtm.

File: TInf.g4 (hunk 1 above)
Comment (ins305) [reply TInf.g4]: Sorry, my mistake. I changed it back now.

**Review by marcauberer — APPROVED** [review]
lgtm, thanks!

**Pull request merged**